### ISSUE TITLE
Seems like gcc pch is broken with Travis

### ISSUE THREAD

**Issue by afbjorklund** [ccache/ccache#217]
e.g. https://travis-ci.org/ccache/ccache/jobs/333614562

```
Running test suite pch.....pch.c:1:17: error: one or more PCH files were found, but they were invalid
 #include "pch.h"
                 ^
pch.c:1:17: error: use -Winvalid-pch for more information
pch.c:1:17: fatal error: pch.h: No such file or directory
compilation terminated.
```

**Comment by jrosdahl** [comment]
Fixed in 958c4c2.

**Comment by opoplawski** [comment]
I seem to be seeing this while building ccache 3.7.7 on RHEL8, but not on Fedora or RHEL7:
```
Running test suite pch............cc1: error: one or more PCH files were found, but they were invalid
cc1: error: use -Winvalid-pch for more information
cc1: fatal error: dir/pch.h: No such file or directory
compilation terminated.
FAILED
Test suite:     pch
Test case:      Use .gch, -include, other dir for .gch
Failure reason: Expected "cache miss" to be 1, actual 0
ccache -s:
cache directory                     /builddir/build/BUILD/ccache-3.7.7/testdir.159309/.ccache
primary config                      /builddir/build/BUILD/ccache-3.7.7/testdir.159309/ccache.conf
secondary config      (readonly)    
stats updated                       Thu Feb 20 03:25:55 2020
cache hit (direct)                     0
cache hit (preprocessed)               0
cache miss                             0
cache hit rate                      0.00 %
preprocessor error                     1
cleanups performed                     0
files in cache                         0
cache size                           0.0 kB
Test data and log file have been left in testdir.159309
```

**Comment by jrosdahl** [comment]
@opoplawski wrote:
> I seem to be seeing this while building ccache 3.7.7 on RHEL8, but not on Fedora or RHEL7:

Addressed in 0d06da7 (master) and b2651f3 (3.7-maint).

**Comment by opoplawski** [comment]
That seems to have worked.  Thanks!
